controllers/discovery/searchUser.js

```diff
@@ -89,7 +90,8 @@ const SearchUser = async (req, res) => {
           AND u.is_banned = false 
           AND u.verified_status != 'UNVERIFIED') 
       ORDER BY  
-          is_followed DESC,
+          recently_active DESC,
+          community_info DESC,
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The updated sorting order prioritizes users based on their recent activity, community information, karma score, and follower count. This is a significant change from the previous sorting order which prioritized followed users, karma score, and follower count. Ensure that this change aligns with the desired user experience and business requirements.
```diff
- ORDER BY  
- is_followed DESC,
- u.karma_score DESC,
- followersCount DESC
+ ORDER BY  
+ recently_active DESC,
+ community_info DESC,
+ u.karma_score DESC,
+ followersCount DESC
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->